frame/evm/precompile/contracts/issuing/Cargo.toml

```diff
@@ -0,0 +1,57 @@
+[package]
+authors     = ["Darwinia Network <hello@darwinia.network>"]
+description = "Issuing precompile for EVM pallet."
+edition     = "2018"
+homepage    = "https://darwinia.network/"
+license     = "GPL-3.0"
+name        = "darwinia-evm-precompile-issuing"
+readme      = "README.md"
+repository  = "https://github.com/darwinia-network/darwinia-common/"
+version     = "2.1.0"
+
+[dependencies]
+# crates
+codec                    = { default-features = false, package = "parity-scale-codec", version = "2.0.1" }
+dp-evm  = { default-features = false, path = "../../../../../primitives/evm" }
+evm                     = { version = "0.24.0", default-features = false, features = ["with-codec"] }
+ripemd160                = { default-features = false, version = "0.9.1" }
```

Comment:
Please remove these useless dependencies and keep `*.toml` clean.  such as `ripemd160`

Reply:
Please check not only this file but also other *.toml files.